app/src/main/java/co/condorlabs/emerald/theme/Theme.kt

```diff
@@ -0,0 +1,36 @@
+package co.condorlabs.emerald.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val DarkColorPalette = darkColors(
```

Comment:
in wallet we do not use DarkPalette, please delete it

Reply:
Done